lib/music/music_album.rb

```diff
@@ -0,0 +1,7 @@
+class MusicAlbum
+  attr_accessor :on_spotify
+
+  def initiliaze(on_spotify)
+    @on_spotify = on_spotify
+  end
+end
```

Comment:
- [ ] [Optional] You can change this in your next Card Task, but I'd suggest making this class inherit Item class. 

Tip: relative path for getting item class rn: './.././../item.rb'.